src/platform/forms-system/src/js/web-component-patterns/datePatterns.jsx

```diff
@@ -43,6 +43,13 @@ const currentOrPastDateUI = options => {
     'ui:options': {
       ...uiOptions,
     },
+    'ui:confirmationData': ({ formData }) => ({
```

Comment:
Here's where I think if we wanted to update it, wouldn't we want to update it on the review page as well? so they are consistent. 
If we do add a new prop, I think it's not a terrible idea to gate it with a feature toggle to draw the connection. e.g. `...(feature ? { 'ui:newProp': () => {} } : {})`

Reply:
Is there an easy way to get the feature flags state into this pattern?